Source/Urho3D/Graphics/LightBakingSettings.h

```diff
@@ -0,0 +1,231 @@
+    unsigned maxSamples_{ 10 };
+    /// Max number of bounces.
+    unsigned maxBounces_{ 2 };
+    /// Position bias in direction of face normal after hit. Scaled with position.
+    float scaledPositionBounceBias_{ 0.00002f };
+    /// Constant position bias in direction of face normal after hit.
+    float constPositionBounceBias_{ 0.0f };
+};
+
+/// Parameters for indirect light filtering.
+struct EdgeStoppingGaussFilterParameters
+{
+    /// Kernel radius.
+    int kernelRadius_{ 2 };
+    /// Upscale factor for offsets.
+    int upscale_{ 1 };
+    /// Color weight. The lesser value is, the more color details are preserved on flat surface.
+    float luminanceSigma_{ 10.0f };
+    /// Normal weight. The higher value is, the more color details are preserved on normal edges.
+    float normalPower_{ 4.0f };
+    /// Position weight. The lesser value is, the more color details are preserved on position edges.
+    float positionSigma_{ 1.0f };
+};
+
+/// Lightmap stitching settings.
+struct LightmapStitchingSettings
+{
+    /// Number of iterations.
+    unsigned numIterations_{ 8 };
+    /// Blend factor.
+    float blendFactor_{ 0.5f };
+    /// Render path used for stitching.
+    ea::string renderPathName_{ "RenderPaths/Forward.xml" };
+    /// Model used for background during stitching.
+    ea::string stitchBackgroundModelName_{ "Models/Plane.mdl" };
+    /// Technique used for background during stitching.
+    ea::string stitchBackgroundTechniqueName_{ "Techniques/DiffUnlit.xml" };
+    /// Technique used for seams rendering during stitching.
+    ea::string stitchSeamsTechniqueName_{ "Techniques/DiffUnlitAlpha.xml" };
+};
+
+/// Light calculation properties that can be used to adjust result.
+struct LightCalculationProperties
+{
+    /// Emission light brightness multiplier.
+    float emissionBrightness_{ 1.0f };
+    /// Background light color.
+    Vector3 backgroundColor_{};
+    /// Background light brightness multiplier.
+    float backgroundBrightness_{ 1.0f };
+    /// Background cubemap.
+    SharedPtr<ImageCube> backgroundImage_{};
+};
+
+/// Incremental light baker settings.
+struct IncrementalLightBakerSettings
+{
+    /// Size of the chunk.
+    Vector3 chunkSize_ = Vector3::ONE * 128.0f;
+    /// Additional space around chunk to collect indirect lighting.
+    float indirectPadding_ = 32.0f;
+    /// Shadow casting distance for directional light.
+    float directionalLightShadowDistance_ = 128.0f;
+    /// Output directory name.
+    ea::string outputDirectory_;
+    /// Global illumination data file.
+    ea::string giDataFileName_{ "GI.bin" };
+    /// Lightmap name prefix.
+    ea::string lightmapNamePrefix_{ "Textures/Lightmap-" };
+    /// Lightmap name suffix.
+    ea::string lightmapNameSuffix_{ ".png" };
+    /// Light probe group name prefix.
+    ea::string lightProbeGroupNamePrefix_{ "Binary/LightProbeGroup-" };
+    /// Light probe group name suffix.
+    ea::string lightProbeGroupNameSuffix_{ ".bin" };
```

Comment:
Prefix and suffix strings could be replaced with a single string that defines format. Like `ea::string lightProbeGroupNameFormat_{"Binary/LightProbeGroup-{}.bin"};` which you can use as `Format(lightProbeGroupNameFormat_, ...)` later.